src/js/Attachment/__test__/on.test.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import Attachment from '../index';
+
+describe('Unit test Attachment on', () => {
+  beforeEach(() => {
+    jest.spyOn(console, 'error');
+    // @ts-ignore
+    console.error.mockImplementation(() => { });
+  });
+  afterEach(() => {
+    // @ts-ignore
+    console.error.mockRestore();
+  });
+
+  test('on is called successfully', () => {
+    const myAttachment = new Attachment();
+    myAttachment.on('filesAdd', () => {});
+    myAttachment.on('fileRemove', () => {});
```

Comment:
Miss some expectation.
Need to check this function is called when file is added/removed

Reply:
This is to verify that `on` is called successfully to assign event handler. operation of `_onFilesAdd` and `_onFileRemove` are tested in other test suit.